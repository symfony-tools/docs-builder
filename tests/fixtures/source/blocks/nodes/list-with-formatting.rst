* List **item 1**
* ``List`` item 2
* *List* item 3
* List item 4

#. List **item 1**
#. ``List`` item 2
#. *List* item 3
#. List item 4

**Quis:** ``int[]``
    Exercitation ullamco *laboris nisi* ut aliquip ex ea commodo consequat.

``Sed do eiusmod``
    Tempor ``incididunt`` ut *labore* et dolore magna aliqua. Ut enim ad minim veniam.

Duis *aute irure*
    Dolor in **reprehenderit** in voluptate velit esse.
