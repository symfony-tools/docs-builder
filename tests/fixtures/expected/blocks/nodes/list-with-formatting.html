<ul>
    <li>List <strong>item 1</strong></li>
    <li><code translate="no" class="notranslate">List</code> item 2</li>
    <li><em>List</em> item 3</li>
    <li>List item 4</li>
</ul>

<ol class="arabic">
    <li>List <strong>item 1</strong></li>
    <li><code translate="no" class="notranslate">List</code> item 2</li>
    <li><em>List</em> item 3</li>
    <li>List item 4</li>
</ol>

<dl>
    <dt><strong>Quis:</strong> <code translate="no" class="notranslate">int[]</code></dt>
    <dd>Exercitation ullamco <em>laboris nisi</em> ut aliquip ex ea commodo consequat.</dd>

    <dt><code translate="no" class="notranslate">Sed do eiusmod</code></dt>
    <dd> Tempor <code translate="no" class="notranslate">incididunt</code> ut <em>labore</em> et dolore magna aliqua. Ut enim ad minim veniam. </dd>

    <dt>Duis <em>aute irure</em></dt>
    <dd> Dolor in <strong>reprehenderit</strong> in voluptate velit esse. </dd>
</dl>
